### ISSUE TITLE
Training/Testing/Validation Set Split

### ISSUE THREAD

**Issue by malineha** [dr-pato/audio_visual_speech_enhancement#8]
hello Sir,
In Facelandmark research Paper
The resulting dataset was split into disjoint sets of 25/4/4 speakers for training/validation/testing respectively.

But In my case ,I am Working On dataset of only 2 Speakers 
in which each speaker have 1000 records(1000audio,1000video)

how can I split these records in Training/Testing/Validation set
I am confused plz help me...
I also confused about Mix speech training/validation/testing split according to above dataset...
Plz help me

**Comment by dr-pato** [comment]
Hi,
I trained a speker independent system so I need to have no overlap between train/validation/test splits. It is not very reasonable to train a model with two speakers only..maybe you can train models with a speaker dependent setting, that will also works if you have enough training data.

Giovanni

**Comment by malineha** [comment]
sorry sir...it was my mistake...I need to change my dataset..
